cpp/src/io/json/write_json.cu

```diff
@@ -25,33 +25,36 @@
 #include <cudf/column/column_device_view.cuh>
 #include <cudf/column/column_factories.hpp>
 #include <cudf/detail/copy.hpp>
+#include <cudf/detail/iterator.cuh>
```

Comment:
Are all of these headers still needed? Offhand I see that `cudf/detail/valid_if.cuh` is not being used in this file as far as I can tell.

Reply:
did a include cleanup once again.